Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -113,6 +113,22 @@ extension MongoClient {
 
         return true
     }
+
+    internal convenience init(options: ClientOptions? = nil) throws {
+        try self.init(connectionString: getConnStr(), options: options)
+    }
+
+    internal convenience init() throws {
+        try self.init(connectionString: getConnStr())
+    }
+}
+
+func getConnStr() -> String {
```

Comment:
Let's 
1) Move this into the `XCTestCase` extension above, next to `getSpecsPath`. Eventually, we're going to [create a base test class](https://jira.mongodb.org/browse/SWIFT-207), which will inherit from `XCTestCase` and be a natural place for methods such as this and `getSpecsPath`, and `is32Bit` to live. In the meantime, let's keep them all together, and can you please add a TODO above the extension noting its contents will be moved into the base test class in SWIFT-207?

2) Rewrite this as a computed property called `connStr`. Generally a computed property named `x` is more Swift-y than a `getX` method... which I did not realize back when I wrote that method. 🙂 If you feel like it, you could also rename `getSpecsPath` to `specsPath` and fix that wherever it's used. Or we can just do it later as part of SWIFT-207. :)

Reply:
So I've added the `TODO` comment, but the other ideas related to moving the function into `XCTestCase` would mean that the convenience initializers no longer have the ability to get the connection string.

We can of course fix this, by either changing the signature to take an XCTestCase (seems pretty weird) or change all instances where we initialize a `MongoClient` to explicitly specify a connection string by using the computed property.

Honestly, I don't think this is a _bad_ thing, but I just figured I'd hold off on this in case this changes what we want to do with `getConnStr()`, since this change probably has ramifications outside of `TestUtils.swift`.

Reply:
Sounds brilliant. Done (both for `getConnStr()` -> `connStr` and `getSpecsPath()` -> `specsPath`).